test/test_query.py

```diff
@@ -772,6 +772,38 @@ def test_match_after_set_none(self):
         matched = self.lib.items(NoneQuery(u'rg_track_gain'))
         self.assertInResult(item, matched)
 
+    def test_match_slow(self):
+        item = self.add_item()
+        matched = self.lib.items(NoneQuery(u'rg_track_peak', fast=False))
+        self.assertInResult(item, matched)
+
+    def test_match_slow_after_set_none(self):
+        item = self.add_item(rg_track_gain=0)
+        matched = self.lib.items(NoneQuery(u'rg_track_gain', fast=False))
+        self.assertNotInResult(item, matched)
+
+        item['rg_track_gain'] = None
+        item.store()
+        matched = self.lib.items(NoneQuery(u'rg_track_gain', fast=False))
+        self.assertInResult(item, matched)
+
+    def test_query_repr(self):
+        fld = u'rg_track_gain'
+        if sys.version_info <= (3, 0):
+            self.assertEquals("NoneQuery(u'{}', True)".format(fld),
+                              str(NoneQuery(fld)))
+            self.assertEquals("NoneQuery(u'{}', True)".format(fld),
+                              str(NoneQuery(fld, fast=True)))
+            self.assertEquals("NoneQuery(u'{}', False)".format(fld),
+                              str(NoneQuery(fld, fast=False)))
+        else:
+            self.assertEquals("NoneQuery('{}', True)".format(fld),
+                              str(NoneQuery(fld)))
+            self.assertEquals("NoneQuery('{}', True)".format(fld),
+                              str(NoneQuery(fld, fast=True)))
+            self.assertEquals("NoneQuery('{}', False)".format(fld),
+                              str(NoneQuery(fld, fast=False)))
+
```

Comment:
I am too n00b to Python to understand what is the story with unicode  there. This is not really part of the issue but I though with this we have all methods of the class covered. Can you pls have a look?

Reply:
TBH I think we can just remove the tests for `repr`… the problem is that Python 2 string literals used syntax like `u'foo'` while on Python 3 they are just `'foo'`. Fortunately, `repr` formatting is so standard that IMO testing it is sort of overkill.